src/app/containers/Ad/__snapshots__/index.test.jsx.snap

```diff
@@ -22,6 +22,7 @@ exports[`Ad Container Snapshots should correctly render a Canonical ad 1`] = `
 <div>
       <div
         class="c0"
+        tabindex="-1"
```

Comment:
Should this be aria-hidden true instead?

Reply:
Yes, snapshots are updated now